packages/react-icons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spnels-fluent-system-test0000",
+  "version": "1.1.50",
+  "description": "Fluent System Icons are a collection of familiar, friendly, and modern icons from Microsoft.",
+  "author": "Microsoft Inc.",
```

Comment:
Do you need this author tag? I see that it's been removed on a separate PR here: https://github.com/microsoft/fluentui-system-icons/pull/177